python/kfserving/kfserving/server.py

```diff
@@ -11,7 +11,7 @@
 DEFAULT_GRPC_PORT = 8081
 DEFAULT_PROTOCOL = "tensorflow/HTTP"
 
-parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(add_help=False)
```

Comment:
Can we set this to true?

Reply:
This doesn't work for me 
```
python -m xgbserver
Traceback (most recent call last):
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/clive/go/src/github.com/kubeflow/kfserving/python/xgbserver/xgbserver/__main__.py", line 9, in <module>
    parser = argparse.ArgumentParser(parents=[kfserving.server.parser])
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/argparse.py", line 1672, in __init__
    self._add_container_actions(parent)
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/argparse.py", line 1441, in _add_container_actions
    group_map.get(action, self)._add_action(action)
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/argparse.py", line 1571, in _add_action
    action = super(_ArgumentGroup, self)._add_action(action)
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/argparse.py", line 1381, in _add_action
    self._check_conflict(action)
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/argparse.py", line 1520, in _check_conflict
    conflict_handler(action, confl_optionals)
  File "/home/clive/anaconda3/envs/kfserving/lib/python3.7/argparse.py", line 1529, in _handle_conflict_error
    raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument -h/--help: conflicting option strings: -h, --help
```

`False` makes sense to me as we assume child args processors will create the help.